job_templates/ansible_roles_-_install_from_git.erb

```diff
@@ -15,6 +15,11 @@ template_inputs:
   description: For example, '/etc/ansible/roles/foobar' . Look at '/etc/ansible/ansible.cfg'
     roles_path option to find what is your roles_path
   advanced: false
+- name: force
+  required: false
+  input_type: user
+  description: If yes, any modified files in the working repository will be discarded.
+  advanced: true
```

Comment:
Probably would be also good to specify options yes,no. I think if you set no as first, it will be default value

Reply:
ah yes, forgot those values. However, from looking in the UI, there doesn't seem to be a way to set a default value (which is a shame :-( ), this is what caused me to add a fallback value to the input('force') below.

But perhaps, there's a way to set a default out of my knowledge?

Reply:
There's no way to set default and yes, it's a shame :-) the only reasonable option is to specify options, with first being the preselected one. I moved the [tracking issue](https://projects.theforeman.org/issues/11367) from REX to core, it shouldn't be too complicated to add.

Reply:
Hey @ares! Great, will look into that one first and come back to this once merged, if that is okay for you?

Reply:
That would be awesome, feel free to ping me in that PR directly :-)

Reply:
@UXabre, the default value was added in Foreman 1.24, I think this was the last thing to finish this PR, could you please add that? since `input('force')` will now always contain the value, the comment below still holds

Reply:
Allright! Let me read up on this default value and finish this template! Thanks for following up!